Dockerfile

```
@@ -108,6 +108,10 @@ COPY opt/aiidalab-singleuser /opt/
 COPY opt/prepare-aiidalab.sh /opt/
 COPY my_init.d/prepare-aiidalab.sh /etc/my_init.d/80_prepare-aiidalab.sh
 
+# Get aiidalab-home app.
+RUN git clone https://github.com/aiidalab/aiidalab-home
```

One more comment: I suggest to slightly update the `clone` command here and in the future we should also specify a specific *tagged (pre-)release* of aiidalab-home.
```suggestion
RUN git clone --branch='master' --single-branch --depth=1 https://github.com/aiidalab/aiidalab-home
```

---

> I suggest to slightly update the `clone` command here and in the future we should also specify a specific _tagged (pre-)release_ of aiidalab-home.

agreed!